sys/auto_init/auto_init.c

```diff
@@ -277,9 +277,14 @@ void auto_init(void)
 #endif
 
 #ifdef MODULE_SX127X
+#ifdef MODULE_SEMTECH_LORAMAC
```

Comment:
IMO subordinate auto_init_loramac() to MODULE_SX127X is wrong, you are mixing up the mac layer and the network interface

Reply:
I don't see why this should be in GRNC_NETIF, it has nothing to do with GNRC.